idea/resources-en/intentionDescriptions/HLAddNamesToFollowingArgumentsIntention/description.html

```diff
@@ -0,0 +1,6 @@
+<html>
+<body>
+This intention adds names to all positional arguments of a function call according to the named argument syntax,
+starting with the current argument.
```

Comment:
Then, maybe adding a test where `<spot>` is on the first argument, 24?

Reply:
This is covered in https://github.com/JetBrains/intellij-kotlin/blob/793c18b9d1db0c2d43b4ff53224a632463c43a4f/idea/tests/testData/intentions/addNamesToFollowingArguments/first.kt

`HLAddNamesToFollowingArgumentsIntention` is _not_ applicable when the caret is on the first argument, because this is equivalent to `HLAddNamesToCallArgumentsIntention` (adding names to all arguments).